src/SimpleGraphs/Generators/staticgraphs.jl

```diff
@@ -270,6 +270,27 @@ function _binary_tree(k::T) where {T<:Integer}
     return SimpleGraph{T}(ne, fadjlist)
 end
 
+function _binary_tree_digraph(k::T) where {T<:Integer}
+    k <= 0 && return SimpleDiGraph{T}(0)
+    k == 1 && return SimpleDiGraph{T}(1)
+    n = T(2^k - 1)
+
+    ne = Int(n - 1)
+    fadjlist = [Vector{T}() for _ = one(T):n]
+    badjlist = [Vector{T}() for _ = one(T):n]
+    @inbounds fadjlist[1] = T[2, 3]
+    @inbounds for i in 1:(k - 2)
+        @simd for j in (2^i):(2^(i + 1) - 1)
```

Comment:
maybe a comment here on `(2^i):(2^(i + 1) - 1)`

Reply:
That's a good idea.